packages/codemods/src/codemods/v6.x.x/replaceComponentSvgs/replaceComponentSvgs.ts

```diff
@@ -17,6 +26,7 @@ async function convertSvgToReactComponent(
     {
       jsxRuntime: 'automatic',
       plugins: ['@svgr/plugin-jsx'],
+      typescript,
```

Comment:
This, together with the `const isTS =` code below is the main change for this PR.
The rest is mostly changes to make the code easier to read and understand